### PR TITLE
Add in ACi support in ntp.conf #207

### PR DESCRIPTION
**Pull request by B1tem3250**
# Description

This adds in the DSCP value for ACI network support.

## Issues Resolved

issue #207 

## Check List

- [x] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by damacus** [comment]
Nice! 

Thanks for the small constrained PR ❤️

**Review by damacus — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ This file is used to list changes made in each version of the ntp cookbook.
 
 ## Unreleased
 
+## 3.11.1 - *2021-11-08*
```
Comment (Xorima) [inline CHANGELOG.md:7]: Hey,

Can you leave this out and just put things under `## Unreleased` the automated system will take care of the rest

**Review by Xorima — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (B1tem3250) [reply CHANGELOG.md:7]: Hi I will do so, sorry never about that.

Comment (B1tem3250) [reply CHANGELOG.md]: I have submitted the change, please let me know if that is correct.

**Review by bmhughes — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bmhughes) [reply CHANGELOG.md]: Yep, that's it!

**Comment by mvangoor** [comment]
I understand this is something Cisco ACI needs, but it is not cisco ACI only (https://manpages.debian.org/testing/ntp/ntp.conf.5.en.html).

I'd favour making this a generic `dscp` option and making a remark in the documentation that cisco ACI needs it.

File: README.md [hunk 2]
```diff
@@ -166,6 +166,10 @@ This cookbook is maintained by the Sous Chefs. The Sous Chefs are a community of
 
   - Number. Defaults to 5, recommended value for stratum is 2 more than the worst-case externally-reachable source of time
 
+- `ntp['aci_support']`
```
Comment (mvangoor) [inline README.md:169]: Please rename this to `ntp['dscp']`

File: attributes/default.rb [hunk 3]
```diff
@@ -57,6 +57,7 @@
 default['ntp']['keys'] = nil
 default['ntp']['trustedkey'] = nil
 default['ntp']['requestkey'] = nil
+default['ntp']['aci_support'] = nil
```
Comment (mvangoor) [inline attributes/default.rb:60]: Please rename this to `ntp['dscp']` as it is a more generic option than cisco aci only

File: README.md (hunk 2 above)
Comment (B1tem3250) [reply README.md]: No Problem, I have done so.

File: attributes/default.rb (hunk 3 above)
Comment (B1tem3250) [reply attributes/default.rb]: Thanks for pointing this out, I have done so.

**Review by mvangoor — APPROVED** [review]
Thanks for the change and adding this option!

**Comment by damacus** [comment]
Putting this on hold for a bit whilst I look at our webhook issues.

**Comment by kitchen-porter** [comment]
Released as: [3.12.0](https://github.com/sous-chefs/ntp/releases/tag/3.12.0)

**Comment by B1tem3250** [comment]
Thank you for all the support and assistance, really appreciate it.
Thanks for merging, I have tested on our side and all is 100%

**Pull request merged**